config/crd/v1beta1/batch.volcano.sh_jobs.yaml

```diff
@@ -50,6 +50,10 @@ spec:
                 to the summary of tasks' replicas
               format: int32
               type: integer
+            minSuccess:
```

Comment:
Suggent to add a validation to this field in the admission

Reply:
done